src/systems/media-frames.js

```diff
@@ -250,8 +250,13 @@ AFRAME.registerComponent("media-frame", {
 
   snapObject(capturedEl) {
     // TODO this assumes media frames are all in world space
-    capturedEl.object3D.position.copy(this.el.object3D.position);
-    capturedEl.object3D.rotation.copy(this.el.object3D.rotation);
+    const worldPosition = new THREE.Vector3();
+    this.el.object3D.getWorldPosition(worldPosition);
+    capturedEl.object3D.position.copy(worldPosition);
+    const worldQuat = new THREE.Quaternion();
+    this.el.object3D.updateWorldMatrix(true);
```

Comment:
You should not need to manually call this, and in fact we don't want to because it will cause extra work. See https://github.com/mozilla/hubs/blob/master/src/utils/threejs-world-update.js#L22. Also worth reading through these rules and this file in general if you havent yet. Not following these rules is source of most of our hubs-specific threejs issues... I do hope we can get rid of it some day, but when we implemented it there was a big performance win, especially due to all the extra Object3Ds aframe introduces into the scene graph.

Reply:
Humm, I didn't know about that file. I've remove the `updateWorldMatrix` call.